dbcon/joblist/tupleaggregatestep.cpp

```diff
@@ -1507,6 +1522,7 @@ void TupleAggregateStep::prep1PhaseAggregate(
             precisionAgg.push_back(0);
             precisionAgg.push_back(0);
             typeAgg.push_back(CalpontSystemCatalog::UBIGINT);
+            csNumAgg.push_back(8);
```

Comment:
This 8 is a a migic to tell aggregate code the output will be a numerical scalar that doesn't depend on charset?

Reply:
8 is the magic number for latin1_swedish_ci, so yes. Something needs to be there to take the space. It will not be referenced in the current implementation.